app/routes.js

```diff
@@ -262,6 +262,42 @@ export const onOtherRouteEnter = (api) => (nextState, replace) => {
   }
 }
 
+export const onSetUserToken = (api, next) => (nextState, replace, cb) => {
+  const TOKEN_LOCAL_KEY = 'authToken';
+
+  if (!api.user.isAuthenticated()) {
+    let localStore = window.localStorage;
```

Comment:
@ysdw  is it secure enough to store the token in the localstorage ?

Reply:
It is the same thing as clicking "Remember Me" case when logged-in.
I use in fact the dom-storage existing value to minimize the code impact on Blip.
I would have prefer to do otherwise, but the impact would have been huge.